herbert_core/admin/@opt_config_manager/private/set_pc_specific_config_.m

```diff
@@ -0,0 +1,9 @@
+function obj = set_pc_specific_config_(obj,config_data,current_pc)
```

Comment:
there's no need to pass `config_data` as an argument because it's an attribute of the `opt_config_manager` object.

Reply:
fixed